### DISCUSSION TITLE
Special font for pictograms

### DISCUSSION THREAD

**Discussion by SpiderBlocker** [PE5PVB/TEF6686_ESP32#594] [Ideas]
In order to display all pictograms and RDS-logo with ani-aliasing, would it be feasable to create a new special font containing all of these items in order to display them with anti-aliasing, just like all other text is displayed throughout the program?

**Comment by PE5PVB** [comment]
Well, the pictograms are not fonts but 1-color images. I can't smooth them.

**Reply by SpiderBlocker** [reply]
Of course, the pictograms are currently no fonts, that is obvious. 
Therefore my idea is to create a new font (eg. similar to Wingdings, that also contains pictograms as "letters") and draw/import all required symbols/logo's into it, which could then be displayed likewise to any of the other fonts already implemented in the program. This could for example be accomplished using FontForge or similar software.
